bundler/src/main.rs

```diff
@@ -454,3 +465,8 @@ fn is_id_start(c: char) -> bool {
 fn is_id_continue(c: char) -> bool {
     is_xid_continue(c) || c == '_' || c == '-'
 }
+
+// Check that a license is any version of CC-BY, CC-BY-SA, or CC0.
+fn is_allowed_cc(license: LicenseId) -> bool {
+    regex!(r"^CC(-(BY(-SA)?)|0)-[0-9]\.[0-9](-[A-Z]+)?$").is_match(license.name)
```

Comment:
`^CC(-BY|-BY-SA|0)-[0-9]\.[0-9](-[A-Z]+)?$` would be a little easier to read imo. I think your regex also has one unnecessary paren group at the start around `BY(-SA)?`.